frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/artifacts/ArtifactVisualization.tsx

```diff
@@ -46,18 +46,18 @@ export const ArtifactVisualization: React.FC<ArtifactVisualizationProps> = ({ ar
     if (artifactType === ArtifactType.MARKDOWN || artifactType === ArtifactType.HTML) {
       const uri = memoizedArtifact.getUri();
       if (uri) {
-        const downloadArtifact = async () => {
-          await getStorageObjectUrl(memoizedArtifact)
-            .then((url) => setDownloadedArtifactUrl(url))
+        const renderArtifact = async () => {
+          await getStorageObjectRenderUrl(memoizedArtifact)
```

Comment:
this should be download

Reply:
The download URL will trigger an automatic download when you put it into an iframe, only the render URL can successfully render the HTML/Markdown resources.